apps/frontend/src/context/UserContext.tsx

```diff
@@ -73,6 +74,16 @@ export const UserContextProvider = ({
     setUser(user);
   };
 
+  const deleteLocalWorkout = (workoutId: string) => {
+    setLocalWorkouts((localWorkouts) => {
+      const filteredWorkouts = localWorkouts.filter(
+        (workout) => workout.id !== workoutId
+      );
+      localStorage['workouts'] = JSON.stringify(filteredWorkouts);
+      return JSON.parse(localStorage['workouts']);
```

Comment:
I was thinking that we could just return the filteredWorkouts, but i seen it was done this way in the save-function, so i also did it here